.github/workflows/spellcheck.yml

```diff
@@ -1,6 +1,9 @@
 name: spellcheck 
 on:
   push:
+    paths:
+       - '**.asciidoc'
```

Comment:
Should we also add adoc?

Reply:
I think there is a convention in docgen to use asciidoc only.